src/Login/Login.js

```diff
@@ -43,7 +43,7 @@ class Login extends Component {
 		submitLoginCredentials(this.state)
 			.then((userInfo) => {
 				this.setUserInfo(userInfo)
-				this.props.hideLoginPage(this.state.name);
+				this.props.hideLoginPage(this.state.name)
```

Comment:
Does the login component need to hold on to the user's name and id, or can it just pass that data to the Header?